tools/WindowsCI.psm1

```diff
@@ -0,0 +1,74 @@
+if($PSVersionTable.PSEdition -ne 'Desktop')
```

Comment:
This is code that other that this imperative code to protect against loading in PowerShell core was just moved from CI.psm1